README.md

```diff
@@ -21,7 +21,7 @@ refer to the [NAB publication](http://arxiv.org/abs/1510.03336) or the [NAB
 Whitepaper](https://github.com/numenta/NAB/wiki#nab-whitepaper) in the wiki for
 more details about NAB scoring, data, motivation, etc.
 
-Please cite the following publication when referring to NAB:
+We encourage you to publish your results on running NAB, and share them with us at [nab@numenta.org](nab@numenta.org). Please cite the following publication when referring to NAB:
```

Comment:
Suggest the following additional statement:

If an NAB user wants to have their results included in the NAB scoreboard, the user should read some of the details in the NAB Whitepaper and also send an email to nab@numenta.org 

Or something to that effect.